sys-cluster/lmod/files/module.1-8.4.20

```diff
@@ -0,0 +1,391 @@
+.Dd $Mdocdate: December 6 2020 $
+.Dt MODULE 1
```

Comment:
Did you write this file, or find it somewhere? Can it be put into `SRC_URI`?

Reply:
I wrote it, cuz this one is quite important to have good documentation for.

Reply:
This package already seems to come with a man page, 
```
/usr/share/Lmod/share/man
/usr/share/Lmod/share/man/cat1
/usr/share/Lmod/share/man/cat1/module.1
```
It's a bit different than yours, seems more verbose actually. No idea where it's coming from the source, but it seems to be available, so no need to duplicate it? :)